src/main/java/io/github/railroad/localization/L18n.java

```diff
@@ -28,14 +41,25 @@ public static void setLanguage(Language language) {
         loadLanguage();
     }
 
+    /**
+     * Returns the current language of the application.
+     * @return {@link Language} The current language of the application.
+     */
     public static Language getCurrentLanguage() {
         return CURRENT_LANG.getValue();
     }
 
+    /**
+     * Returns the Current Language cache, for use in bindings or listeners.
+     * @return {@link ObjectProperty<Language>} The language cache.
+     */
     public static ObjectProperty<Language> currentLanguageProperty() {
         return CURRENT_LANG;
     }
 
+    /**
+     * Gets the currently selected language from the config, and then attempts to load the .lang file for that language into the cache.
+     */
```

Comment:
This is a bit confusing saying it "gets" the currently selected language. If you're skimming you assume that means it does actually get it, which it doesn't. Possibly just rephrase to something like

```suggestion
    /**
     * Attempts to load the .lang file for the current language into the cache.
     */
```